src/pages/Pathway/Pathway.js

```diff
@@ -1,12 +1,16 @@
-import React from 'react';
+import React, {useContext} from 'react';
 import PageHeader from '../../components/PageHeader/PageHeader';
 import PathwayHeader from '../../img/pathway-header.svg';
 import PathDay from '../../components/PathDay/PathDay';
-import './Pathway.css';
-
+import QuizContext from '../Quiz/QuizContext';
 import resources from '../../data/resources';
+import './Pathway.css';
 
 const Pathway = () => {
+    const quiz_result = useContext(QuizContext);
```

Comment:
if you look at the console.log then you'll see that both `quizResults` and `setQuizResults` got passed in as quiz_result. but we only need the former! so instead what you could do to get specific elements of the QuizContext object instead of the whole thing is to do 
`const { quizResults } = useContext(QuizContext);`